app/src/main/java/com/xmartlabs/template/controller/Controller.java

```diff
@@ -2,8 +2,33 @@
 
 import com.xmartlabs.template.BaseProjectApplication;
 
+import io.reactivex.Completable;
+import io.reactivex.Observable;
+import io.reactivex.Single;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
+
 public abstract class Controller {
   public Controller() {
     BaseProjectApplication.getContext().inject(this);
   }
+
+  protected <T> Single<T> makeIoCall(Single<T> serviceCall) {
```

Comment:
maybe you can rename the parameter to `ioCall` to make it match the function name